modulefiles/module_base.jet

```diff
@@ -1,32 +1,29 @@
+#%Module######################################################################
 ##
-##      nems prerequisites
+##      FV3GFS prerequisites
 ##
 
-proc ModulesHelp {} {
-     puts stderr "The prerequisites for compiling or running FV3 on Jet. "
-}
-
-module load newdefaults intel/15.0.3.187 impi/5.1.3.181 szip hdf5 netcdf4/4.2.1.1
-
-module use /lfs3/projects/hfv3gfs/nwprod/lib/modulefiles
-module load bacio/v2.0.2
-module load sp/v2.0.2
-module load ip/v2.0.0
-module load w3nco/v2.0.6
-module load w3emc/v2.2.0
-module load nemsio/v2.2.2
-
-#set NCEPLIBS $::env(NCEPLIBS)
-#module use $NCEPLIBS/modulefiles
-#module load esmf/7.1.0r_impi
-
-module use /lfs3/projects/hwrf-vd/soft/modulefiles
-module load prod_util
-module load grib_util
+module load intel/18.0.5.274
+module load impi/2018.4.274
+module load wgrib2/2.0.8
 module load hpss
-
-# mpiserial on Jet includes a cfp wrapper
-module load mpiserial
+module load nco/4.9.1
+module load cdo/1.9.5
 
 module load gempak/7.4.2
 
+# Load from official NCEPLIBS
+module use -a /mnt/lfs4/HFIP/hfv3gfs/nwprod/NCEPLIBS/modulefiles
+module load prod_util/v1.0.18
+module load grib_util/v1.1.1
+module load g2tmpl/1.6.0
+module load crtm/2.3.0
+
+# Add new path for netcdfp  -- fix for netcdf HDF errors
+setenv NCEPLIBS  /lfs4/HFIP/hfv3gfs/gwv/l0701/lib/
+module use -a /mnt/lfs4/HFIP/hfv3gfs/gwv/l0701/lib/modulefiles
+module load esmflocal/8_0_1
+module load netcdfp/4.7.4
```

Comment:
Jet is using hpc-stack for the model.  Please use those.  
See jet related files here:
https://github.com/ufs-community/ufs-weather-model/tree/develop/modulefiles

Reply:
Modification made as requested